libraries/standard/http/utest/http_utest.c

```diff
@@ -536,7 +535,7 @@ void test_Http_InitializeRequestHeaders_Insufficient_Memory()
  */
 void test_Http_AddHeader_Happy_Path()
 {
-    HTTPStatus_t httpStatus = HTTP_INTERNAL_ERROR;
+    HTTPStatus_t httpStatus = HTTP_PARSER_INTERNAL_ERROR;
```

Comment:
My worry is that some of these have nothing to do with http parser. What do you think would be a more appropriate default status?

Reply:
We shouldnt have a default status just for the purpose of testing and initializing. We can probably change these to HTTP_SUCCESS>